doc/tutorials/running_deb_as_serv_vm.rst

```diff
@@ -43,89 +43,93 @@ Use the following instructions to install Debian.
 Validated Versions
 ******************
 
-- **Debian version:** 10.0 (buster)
-- **ACRN hypervisor tag:** acrn-2019w35.1-140000p
-- **Debian Service VM Kernel version:** 4.19.68-84.iot-lts2018-sos
+- **Debian version:** 10.1 (buster)
```

Comment:
The only ISO available from the download area is 10.6. Can we use that here? (In other words, have we validated it?)

Reply:
No bandwidth to verify 10.6.
@NanlinXie  Any direction?
Thanks!

Reply:
@gvancuts , our previous validation is based on 10.1 version. I have created one git issue to track this.
https://github.com/projectacrn/acrn-hypervisor/issues/5408

Reply:
Sounds good, let's discuss how to tackle this problem in there.